src/renderer/components/navbar/Friends.vue

```diff
@@ -92,7 +103,8 @@ const emits = defineEmits<{
 
 const activeIndex = ref(0);
 const accordianActiveIndexes = ref([0, 1, 2]);
-const friendIdStr = ref("");
+const friendId = ref();
+let addFriendDisabled = ref(true);
```

Comment:
try to use `const` when possible, especially for refs, as it's almost always a mistake to reassign refs, as their `.value` property is usually what people intend to change. helps future devs avoid mistakes

Reply:
Makes sense, I'm new to Vue so appreciate the feedback, fixed.